src/utils/vacations-utils.ts

```diff
@@ -76,22 +76,30 @@ export function formatVacationDaysPayload(vacationDays: VacationDays): VacationD
  */
 export const validateVacationRequestDays = (
   requestedDays: number,
-  unspentDays: number
+  unspentDays: number,
+  options?: { isAdmin?: boolean }
 ): { valid: boolean; errorMessage?: string } => {
   if (requestedDays > unspentDays) {
     const formatString = (template: string, values: Record<string, string | number>) =>
       template.replace(/\{(\w+)\}/g, (_, key) => String(values[key] ?? ""));
```

Comment:
We don't need the replace or the regEx to do this, rather we can do it directly using the strings localization library- there should be examples of passing a dynamic value to a string in the code already